pkg/printers/flags_test.go

```diff
@@ -0,0 +1,50 @@
+package printers
+
+import (
+	"testing"
+)
+
+func TestIllegalPackageSourceChecker(t *testing.T) {
```

Comment:
This is good, but from within `pkg/kubectl/cmd` give me a test that attempts to use a printer with an internal type and fails on the correct error.

Reply:
Added a test in pkg/kubectl/cmd